docs/Getting Started.md

```diff
@@ -127,11 +127,11 @@ $ mix firmware.burn -y
 
 ## Nerves Examples
 
-To get up and running quickly, you can check out our collection of example projects:
+To get up and running quickly, you can check out our collection of example projects. Be sure to set your `NERVES_TARGET` environment variable appropriately as the examples use this to determine the target when building firmware. Visit the [Targets Page](targets.html) for more information on what target name to use for the boards that Nerves supports.
 ```
 $ git clone https://github.com/nerves-project/nerves-examples
 $ cd nerves-examples/blinky
-$ mix deps.get && mix firmware
+$ NERVES_TARGET=rpi3 sh -c "mix deps.get && mix firmware"
```

Comment:
I'm concerned that this little "shell magic" might be intimidating to newcomers. As an alternative, how about `NERVES_TARGET=rpi3 mix do deps.get, firmware`. This has the additional benefit of introducing the existence of the `mix do` command, which people may not have encountered yet.